internal/service/alarms/serve.go

```diff
@@ -67,13 +68,22 @@ func Serve() error {
 	}()
 
 	// Get client for hub
-	hubClient, err := k8s_client.NewClientForHub()
+	hubClient, err := k8s.NewClientForHub()
 	if err != nil {
 		return fmt.Errorf("error creating client for hub: %w", err)
 	}
 
+	// Get Resource Types
+	resourceTypes, err := resourceserver.GetResourceTypes(ctx)
+	if err != nil {
+		// TODO: return error - Ignore for now
+		// Mock data for testing
+		resourceTypes = &[]resources.ResourceType{}
+	}
+
+	// Load dictionary
```

Comment:
We should try to keep the top level `serve` function as minimalistic as possible.  Creating clients, initializing structs, starting servers, etc... we should keep the actual work part (i.e., calling APIs, reading/writing to DB, etc...) within separate functions/files.

Reply:
agree, the idea here is to avoid to call this twice. It will be used in the dictionary and in the server as well. Will see what I can do

Reply:
updated